test/lib/random_schema.cc

```diff
@@ -821,22 +821,22 @@ schema_ptr build_random_schema(uint32_t seed, random_schema_specification& spec)
     auto pk_columns = spec.partition_key_columns(engine);
     SCYLLA_ASSERT(!pk_columns.empty()); // Let's not pull in boost::test here
     for (size_t pk = 0; pk < pk_columns.size(); ++pk) {
-        builder.with_column(to_bytes(format("pk{}", pk)), std::move(pk_columns[pk]), column_kind::partition_key);
+        builder.with_column(to_bytes(format("pk{}", pk)), pk_columns[pk], column_kind::partition_key);
```

Comment:
Why?

Reply:
it's unrelated to this change, and should be dropped.